app/components/layout/keyboard_layout/keyboard_layout.js

```diff
@@ -4,51 +4,52 @@
 import React, {PureComponent} from 'react';
 import PropTypes from 'prop-types';
 import {
-    Keyboard,
+    LayoutAnimation,
     Platform,
     StyleSheet,
     View,
 } from 'react-native';
 
 import * as CustomPropTypes from 'app/constants/custom_prop_types';
 
+const KEYBOARD_HIDDEN_STATE = 3;
```

Comment:
Are there any references for where these constants come from? This one in particular sounds like it's from an enum somewhere.

Reply:
this is an enum in the native code